WooCommerce/Classes/ViewRelated/Products/Edit Product/Reviews/ProductReviewsViewController.swift

```diff
@@ -0,0 +1,358 @@
+        transitionToResultsUpdatedState()
+
+        syncingCoordinator.synchronizeFirstPage()
+    }
+}
+
+
+// MARK: - User Interface Initialization
+//
+private extension ProductReviewsViewController {
+
+    /// Setup: View properties
+    ///
+    func configureView() {
+        navigationItem.title = NSLocalizedString(
+            "Reviews",
+            comment: "Title that appears on top of the Product Reviews screen."
+        )
+        view.backgroundColor = .listBackground
+    }
+
+    /// Setup: Sync'ing Coordinator
+    ///
+    func configureSyncingCoordinator() {
+        syncingCoordinator.delegate = self
+    }
+
+    /// Setup: TableView
+    ///
+    func configureTableView() {
+        view.backgroundColor = .listBackground
+        tableView.backgroundColor = .listBackground
+        tableView.refreshControl = refreshControl
+        tableView.dataSource = viewModel.dataSource
+        tableView.delegate = self
+        tableView.tableFooterView = footerSpinnerView
+    }
+
+    /// Setup: ResultsController
+    ///
+    func configureResultsController() {
+        viewModel.configureResultsController(tableView: tableView)
+    }
+
+    /// Setup: TableViewCells
+    ///
+    func configureTableViewCells() {
+        viewModel.configureTableViewCells(tableView: tableView)
+    }
+}
+
+
+// MARK: - Actions
+//
+private extension ProductReviewsViewController {
+
+    @IBAction func pullToRefresh(sender: UIRefreshControl) {
+        // TODO: Analytics M3
+        syncingCoordinator.synchronizeFirstPage {
+            sender.endRefreshing()
+        }
+    }
+}
+
+
+// MARK: - UITableViewDelegate
+extension ProductReviewsViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return viewModel.delegate.tableView?(tableView, heightForHeaderInSection: section) ?? 0
+    }
+
+    func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
+        return viewModel.delegate.tableView?(tableView, estimatedHeightForRowAt: indexPath) ?? 0
+    }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return viewModel.delegate.tableView?(tableView, heightForRowAt: indexPath) ?? 0
+    }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        tableView.deselectRow(at: indexPath, animated: true)
+        viewModel.delegate.didSelectItem(at: indexPath, in: self)
+    }
+
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        viewModel.delegate.tableView(tableView, willDisplay: cell, forRowAt: indexPath, with: syncingCoordinator)
+    }
+}
+
+
+// MARK: - ResultsController
+//
+private extension ProductReviewsViewController {
+
+    /// Refreshes the Results Controller Predicate, and ensures the UI is in Sync.
+    ///
+    func reloadResultsController() {
```

Comment:
I added it because initially, I thought that, exactly like in the review tab, we need to manage the refresh if something change (for example, if while the view was opened, the default site change under the hood). But since this is not the case, I'm going to remove it, I think that this way everything is cleaner. 👍